.eslintrc.json

```diff
@@ -134,5 +133,29 @@
 			"@typescript-eslint/no-unsafe-return": "off",
 			"@typescript-eslint/no-unsafe-call": "off"
 		}
-	}]
+	},
+	{
+		"files": ["src/**/*.ts", "test/**/*.ts"],
+		"rules": {
+			// TODO: Re-enable gradually
```

Comment:
How are we going to keep track of this TODO? Is it worth creating a ticket/assigning an owner?

Reply:
I can create a ticket if you want but just been working through the ESLint as I find time.  The TODO is mostly to mark that these rules aren't here to disable them long term